thoth/storages/graph/janusgraph.py

```diff
@@ -364,6 +364,26 @@ def create_pypi_package_version(self, package_name: str, package_version: str, *
 
         return python_package, has_version, python_package_version
 
+    def unsolved_runtime_environments(self, package_name: str, package_version: str) -> list:
+        """Get Unsolved Runtime Environment which are not connected and attached to python package."""
```

Comment:
> **U**nsolved **R**untime **E**nvironment

No need have it upper case.

> attached to python package

Actually it is "python package version", not only python package.